components/dropdown/demo/api.min.js

```diff
@@ -2439,7 +2509,7 @@ class AuroDropdown extends r {
     /**
      * @private
      */
-    this.runtimeUtils = new AuroLibraryRuntimeUtils();
+    this.runtimeUtils = new AuroLibraryRuntimeUtils$1();
```

Comment:
Same here re: `$1`. Just want to be sure it's a convention we currently use, and other developers are away of it. It appears throughout the PR.

Reply:
This is also a generated file.